test/Swashbuckle.AspNetCore.Newtonsoft.Test/SchemaGenerator/NewtonsoftSchemaGeneratorTests.cs

```diff
@@ -142,6 +142,33 @@ public void GenerateSchema_GeneratesObjectSchema_IfDictionaryTypeHasEnumKey()
             Assert.Equal(new[] { "Value2", "Value4", "Value8" }, schema.Properties.Keys);
         }
 
+        [Fact]
+        public void GenerateSchema_GeneratesObjectSchema_IfDictionaryTypeHasEnumKey_CamelCasePropertyNamesContractResolver()
+        {
+            JsonSerializerSettings settings = new JsonSerializerSettings()
+            {
+                ContractResolver = new CamelCasePropertyNamesContractResolver()
+            };
+
+            var schema = Subject(null, (s) =>
+            {
+                s.ContractResolver = new CamelCasePropertyNamesContractResolver();
+            }).GenerateSchema(typeof(IDictionary<IntEnum, int>), new SchemaRepository());
```

Comment:
Added CamelCasePropertyNamesContractResolver to a copy of an existing test (GenerateSchema_GeneratesObjectSchema_IfDictionaryTypeHasEnumKey)